### PR TITLE
Add mobile menu

### PR DESCRIPTION
**Pull request by bimbolabuari**
# Portfolio Microverse Project  🚀 

 > This PR adds a project about building a portfolio with  HTML, CSS, and JS that demonstrates the knowledge of Semantic HTML, flexbox, background properties, and mobile-first responsiveness, Responsive web design, CSS media query, DOM manipulation, event listener, arrow function, let and const.

## Built With 🙂 

- Semantic HTML & CSS
-  JavaScript
- Linters (Stylelint, Web hint, Eslint, Lighthouse)

## Live Preview  🎉 

[Live Preview Link](https://deploy-preview-10--bimbola-mv-portfolio.netlify.app/)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+function toggleBurger() {
+  navList.classList.toggle('nav-active');
+  navLinks.forEach((link, index) => {
+    if (link.style.animation) {
+      link.style.animation = '';
+    } else {
+      link.style.animation = `navLinkFade 0.5s ease forwards ${index / 2 + 0.4}s`;
+    }
+  });
+  hamburger.classList.toggle('toggle');
```
Comment (MahmoudBakr23) [inline index.js:14]: - [x] Great job you did so far but I'm afraid that you might have missed a very small detail that you should consider.
When the user clicks the hamburger button, the menu opens ✔️ , when the user clicks the X button, the menu closes ✔️ , and when the user clicks one of the menu links (Portfolio, About, Contact), the menu closes ✔️ , but the button doesn't toggle ❌ . Instead, it should get back to the hamburger button!

_Kindly take a look at the two images down below._
### **Your design**
> The button should be the hamburger in this case!
Screenshot

> The button should be the X in this case!
Screenshot1

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @bimbolabuari and @smunozmo 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct 💯 
- Excellent use of js features and DOM manipulation 💯 
- I loved the animations 💯 
- Descriptive PR headline, summary, and commit messages 💯 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shaher-11 — APPROVED** [review]
#### Hello @bimbolabuari,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by bimbolabuari** [comment]
@MahmoudBakr23, Thanks for reviewing my project 👍 
@Shaher-11, Thanks for approving my project 🎉

**Pull request merged**